bootstrap/events/consumer/streams.go

```diff
@@ -14,6 +14,7 @@ import (
 
 const (
 	thingRemove     = "thing.remove"
+	thingConnect    = "policy.create"
 	thingDisconnect = "policy.delete"
```

Comment:
this should be `group.assign` and `group.unassign`  
But I guess we could not use this operation, because adding group within a group also use the same operation